### DISCUSSION TITLE
The 2021 NetBox user survey is open!

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#5955] [General]
We've launched our annual user survey! This brief questionnaire helps the NetBox maintainers gather feedback from the community to help guide the project's long-term development. Your participation is greatly appreciated!

https://forms.gle/3CZ3cBjvZR6TCAAW6

The survey will remain open until March 31. The results will be collected and shared here the following week.

**Update:** The results [have been published](https://github.com/netbox-community/netbox/discussions/6086)!

**Comment by netsandbox** [comment]
You ask about devices (How many devices exist in your NetBox installation?) but not about virtual machines.
Maybe it's also interesting how many virtual machines are managed with NetBox.

**Reply by DanSheps** [reply]
Thanks for the suggestion, where were you 2 days ago.  lol

**Reply by netsandbox** [reply]
Next time you can send me the survey form for a review before making it public :laughing:

**Comment by jeremystretch** [comment]
The survey is now closed with 275 responses. Thanks to everyone who participated! We should have the results published sometime next week.
